src/components/Breadcrumbs/Breadcrumbs.module.css

```diff
@@ -57,6 +57,7 @@
 .breadcrumbs__icon {
   margin-left: var(--eds-size-1);
   margin-right: var(--eds-size-1);
+  fill: var(--eds-theme-color-icon-neutral-strong);
```

Comment:
as of https://github.com/chanzuckerberg/edu-design-system/pull/953, Icon `color` and `size` should be set as props from the component. In `BreadcrumbsItem`, it's already invoking
```
      <Icon
        className={styles['breadcrumbs__icon']}
        color={EdsThemeColorIconNeutralStrong}
        name="chevron-right"
        purpose="decorative"
        size="0.75rem"
      />
```
so the extra `fill` styling isn't needed

Reply:
@anniehu4 More of a higher-level question: What's the reasoning behind using a `color` prop here instead of a custom property in the CSS? Is this more of a "support the existing system" prop?

Reply:
yep, that's the main motivation! It also cleans up the CSS in certain situations (e.g. different colored icons in different banner variants don't need extra CSS). Don't remember if there were other considerations when we first added the prop